packages/modular-scripts/src/build/index.ts

```diff
@@ -0,0 +1,55 @@
+import { paramCase as toParamCase } from 'change-case';
+
+import getModularRoot from '../utils/getModularRoot';
+import actionPreflightCheck from '../utils/actionPreflightCheck';
+import isModularType from '../utils/isModularType';
+import execSync from '../utils/execSync';
+import getLocation from '../utils/getLocation';
+import { setupEnvForDirectory } from '../utils/setupEnv';
+
+async function build(
+  target: string,
+  preserveModules = true,
+  includePrivate = false,
+): Promise<void> {
+  const modularRoot = getModularRoot();
+  const targetPath = await getLocation(target);
+
+  await setupEnvForDirectory(targetPath);
+
+  const targetName = toParamCase(target);
+
+  if (isModularType(targetPath, 'app')) {
+    if (process.env.USE_MODULAR_ESBUILD) {
```

Comment:
Instead of using process.env, can we add a flatgto the build program? `modular build app --experimental-esbuild`?